update_nightly_list.py

```diff
@@ -0,0 +1,79 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim:fenc=utf-8
+#
+# Copyright © 2018 Martine Lenders <m.lenders@fu-berlin.de>
+#
+# Distributed under terms of the MIT license.
```

Comment:
Oops, did not adapt my vim templates for this Repo. Will fix license and copyright notice (and all the python2 stuff) tomorrow.

Reply:
this is done, right?

Reply:
Nope, totally forgot about that... sorry